docs/introduction/managed-resources.md

```diff
@@ -40,49 +41,52 @@ spec:
       name: mysql-secret
       namespace: crossplane-system
   providerConfigRef:
-    name: mycreds
+    name: default
   reclaimPolicy: Delete
 ```
 
 In Kubernetes, `spec` top field represents the desired state of the user.
-Crossplane adheres to that and has its own conventions about how the fields under
-`spec` should look like.
+Crossplane adheres to that and has its own conventions about how the fields
+under `spec` should look like.
 
 * `writeConnectionSecretToRef`: A reference to the secret that you want this
-  managed resource to write its connection secret that you'd be able to mount
-  to your pods in the same namespace. For `RDSInstance`, this secret would contain
+  managed resource to write its connection secret that you'd be able to mount to
+  your pods in the same namespace. For `RDSInstance`, this secret would contain
   `endpoint`, `username` and `password`.
-  
-* `providerConfigRef`: Reference to the `ProviderConfig` resource that will provide information
-  regarding authentication of Crossplane to the provider. `ProviderConfig` resources
-  refer to `Secret` and potentially contain other information regarding authentication.
+
+* `providerConfigRef`: Reference to the `ProviderConfig` resource that will
+  provide information regarding authentication of Crossplane to the provider.
+  `ProviderConfig` resources refer to `Secret` and potentially contain other
+  information regarding authentication. The `providerConfigRef` is defaulted to
+  a `ProviderConfig` named `default` if omitted.
 
 * `reclaimPolicy`: Enum to specify whether the actual cloud resource should be
```

Comment:
I believe this should be updated to `deletionPolicy` and possible values changed to `Delete` and `Orphan`